lisa/transformers/package_installer.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from dataclasses import dataclass, field
+from pathlib import PurePath
+from typing import Any, Dict, List, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.operating_system import RPMDistro
+from lisa.tools import Rpm
+from lisa.transformers.deployment_transformer import (
+    DeploymentTransformer,
+    DeploymentTransformerSchema,
+)
+from lisa.util import UnsupportedDistroException, field_metadata
+
+
+@dataclass_json()
+@dataclass
+class PackageInstallerSchema(DeploymentTransformerSchema):
+    ...
+
+
+class PackageInstaller(DeploymentTransformer):
+    @classmethod
+    def type_name(cls) -> str:
+        return "package_installer"
+
+    def validate(self) -> None:
+        raise NotImplementedError()
+
+
+@dataclass_json()
+@dataclass
+class RPMPackageInstallerTransformerSchema(PackageInstallerSchema):
+    # Node's local absolute path of directory where rpm files are located
+    directory_path: str = field(default="", metadata=field_metadata(required=True))
```

Comment:
"directory" and "path" are similar. Use `path` is enough.

Reply:
`path` might mislead into path of the file. How about just `directory` ?

Reply:
path is shorter. But it's ok to me for `directory`.

Reply:
Thanks. going with `directory`